data/make_data.py

```
@@ -0,0 +1,160 @@
+import json, math, random, sys, time, shutil, os, string, re, fileinput
```

Ruff (0.11.9)

`math` imported but unused

Remove unused import

(F401)